### PR TITLE
Finish The March on Winterfell pack

### PR DESCRIPTION
**Pull request by DukeTax**

### CODE REVIEW

File: packs/TMoW.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "cgdbId": 32,
     "code": "TMoW",
     "name": "The March on Winterfell",
-    "releaseDate": null,
+    "releaseDate": "2016-07-05",
```
Comment (ystros) [inline packs/TMoW.json]: 2018. 😄

File: packs/TMoW.json [hunk 2]
```diff
@@ -75,6 +137,201 @@
...
+            "loyal": false,
+            "cost": 3,
+            "icons": {
+                "military": true,
+                "intrigue": false,
+                "power": true
+            },
+            "strength": 2,
+            "traits": [
+                "IronBorn"
```
Comment (ystros) [inline packs/TMoW.json]: Typo (should be lowercase b)

File: packs/TMoW.json [hunk 3]
```diff
@@ -75,6 +137,201 @@
...
+            ],
+            "text": "Shadow (2).\n<b>Interrupt:</b> When Hagen’s Daughter would be killed, save her and return her to shadows.",
+            "deckLimit": 3
+        },
+        {
+            "code": "11032",
+            "type": "event",
+            "name": "\"Steel Rain\"",
+            "faction": "greyjoy",
+            "loyal": false,
```
Comment (ystros) [inline packs/TMoW.json]: Should be loyal

**Comment by DukeTax** [comment]
Thanks!

**Pull request merged**